client/src/context/Data/DataState.js

```diff
@@ -16,25 +17,27 @@ const DataState = (props) => {
     dispatch({ type: SET_LOADING });
   };
 
-  useEffect(() => {
-    const getData = async () => {
-      setLoading();
-      try {
-        // console.log("Start loading data.");
-        const availableTopics = (await axios.get(`https://puxec9.deta.dev/keys`)).data;  
-        dispatch({ type: GET_DATA, payload: availableTopics });
-      } catch (err) {
-        console.error(err);
-      }
-    };
-    getData();
-  }, []);
+  const hasLoaded = () => {
+    return state.loaded;
+  }
```

Comment:
Instead of creating a function that returns `state.loaded`, we can instead go to `<DataContext.Provider value={{...}}>` at the bottom of the file, and set `hasLoaded: state.loaded`

That way we can delete this redundant function 👍.